src/app/CommandHandler.h

```diff
@@ -374,13 +347,12 @@ class CommandHandler
      * Most applications are likely to use `AddResponseData` as a more convenient
      * one-call that auto-sets command ID and creates the underlying encoders.
      */
-    CHIP_ERROR AddResponseDataViaEncoder(const ConcreteCommandPath & aRequestCommandPath, CommandId commandId,
-                                         EncoderToTLV & encoder)
+    CHIP_ERROR AddResponseData(const ConcreteCommandPath & aRequestCommandPath, CommandId commandId,
```

Comment:
```suggestion
    CHIP_ERROR AddResponseData(const ConcreteCommandPath & aRequestCommandPath, CommandId aResponseCommandId,
```
Two things here:

1. Follow file style for argument naming.
2. Make it clear that this is the command ID of the response command.  It took me a while to understand why we need a command ID separate from the one in aRequestCommandPath.